### PR TITLE
CAL-412 Updated Cloud Cover to be a float

### PR DESCRIPTION
**Pull request by bdeining**
#### What does this PR do?
CAL-412 Updated Cloud Cover to be a float.  Updated NSILI transformer accordingly
#### Who is reviewing it? 
@troymohl @beyelerb 
(please choose AT LEAST two reviewers that need to approve the PR before it can get merged; if a component team is listed, at least one of its members needs to approve)
#### Choose 2 committers to review/merge the PR.
(please choose ONLY two committers from below, delete the rest)
@brendan-hofmann
@jlcsmith
#### How should this be tested?
Full Build
#### What are the relevant tickets?
[CAL-412](https://codice.atlassian.net/browse/CAL-412)
#### Checklist:
- [ ] Documentation Updated
- [x] Update / Add Unit Tests
- [ ] Update / Add Integration Tests

### CODE REVIEW

**Comment by jlcsmith** [comment]
@bdeining - have you checked to see if the imaging code needs to be updated as well?

**Comment by bdeining** [comment]
@jlcsmith updated imagery code
@brendan-hofmann updated docs

File: distribution/docs/src/main/resources/content/_metadataAttributes/NITF_TRE_PIAIMC-table.adoc [hunk 1]
```diff
@@ -19,7 +19,7 @@
 |values 0-100 +
 else null
 |isr.cloud-cover
-|Integer
+|Double
```
Comment (emmberk) [inline distribution/docs/src/main/resources/content/_metadataAttributes/NITF_TRE_PIAIMC-table.adoc]: Does the PIAIMC attribute need to be updated too? 
https://github.com/codice/alliance/blob/436b3705cb6f0981e20e1f0ddd1075cda0ff9e43/catalog/imaging/imaging-transformer-nitf/src/main/java/org/codice/alliance/transformer/nitf/common/PiaimcAttribute.java#L122-L128
https://github.com/codice/alliance/blob/436b3705cb6f0981e20e1f0ddd1075cda0ff9e43/catalog/imaging/imaging-transformer-nitf/src/main/java/org/codice/alliance/transformer/nitf/common/PiaimcAttribute.java#L286-L294

File: distribution/docs/src/main/resources/content/_metadataAttributes/nsili-attributes-table.adoc [hunk 2]
```diff
@@ -304,7 +304,7 @@ else, `id`
 |cloudCoverPercentage
 |`isr.cloud-cover`
 |none
-|Short
+|Double
```
Comment (emmberk) [inline distribution/docs/src/main/resources/content/_metadataAttributes/nsili-attributes-table.adoc]: Does the ISR attribute need to be updated too?
https://github.com/codice/alliance/blob/d559287819156c69ff451b45b67e78f8ac7a94f1/catalog/core/catalog-core-api-impl/src/main/java/org/codice/alliance/catalog/core/api/impl/types/IsrAttributes.java#L32-L39

File: distribution/docs/src/main/resources/content/_metadataAttributes/isr-attributes-table.adoc [hunk 3]
```diff
@@ -89,8 +89,8 @@
 
 |isr.cloud-cover
 |Identifies the percentage of the image that is obscured by clouds.
-|Integer
-|0-100
+|Double
+|0.0-100.0
```
Comment (emmberk) [inline distribution/docs/src/main/resources/content/_metadataAttributes/isr-attributes-table.adoc]: Does the ISR attribute need to be updated too?
https://github.com/codice/alliance/blob/d559287819156c69ff451b45b67e78f8ac7a94f1/catalog/core/catalog-core-api-impl/src/main/java/org/codice/alliance/catalog/core/api/impl/types/IsrAttributes.java#L32-L39

**Comment by bdeining** [comment]
build now

**Comment by troymohl** [comment]
I believe that https://github.com/codice/alliance/blob/master/catalog/nsili/catalog-nsili-common/src/main/java/org/codice/alliance/nsili/common/ResultDAGConverter.java#L769 also needs to be updated as the metacard will no longer have an integer value for Cloud Cover.

**Comment by bdeining** [comment]
@troymohl updated the ResultDAGConverter

**Comment by bdeining** [comment]
build now

**Pull request merged**